### DISCUSSION TITLE
Editing Website Howto

### DISCUSSION THREAD

**Discussion by rufuspollock** [life-itself/community#804] [General]
This discussion is for consolidating questions and (links to) existing howtos and tutorials for editing the website.

Once in a good enough state this could move to lifeitself.org/tao/comms/website or similar.

**Comment by laurenwigmore** [comment]
Questions/How To's raised:
- How can I access the Community Repository?
- How can I add someone e.g. a new person, to a Repository?
- How do I create a new page on the website (with information related to a course)? 
- How do I publish a blog post? 
 - N.B. [The Comms team has an SOP for this already accessible only to LI members](https://docs.google.com/document/d/11pnRk_dbxT3NmoLvZ_IQDnebRm4UCaX2-e3E29t_Ujc/edit?usp=drive_link)
- How can I re-order the website menu bar?

**Comment by rufuspollock** [comment]
@laurenwigmore is there a reason the SOP is private - could we post this to the tao?

**Reply by laurenwigmore** [reply]
No, no reason. We were working to get more completed and then move across and update the Tao accordingly.

**Reply by catherinet1** [reply]
I am following the [Publishing a Blog Post guide](https://docs.google.com/document/d/11pnRk_dbxT3NmoLvZ_IQDnebRm4UCaX2-e3E29t_Ujc/edit) to publish a blog post today. Questions coming up for me are:

- When I fill in the 'categories' element of the frontmatter, how do I know what are eligible categories I can choose from? **What I did here was to go to the LI [blog](https://lifeitself.org/blog) page, and look at the list of tags already in use. But how do we avoid infinite tags being created for very similar things, e.g. currently there is a 'research' tag and 'life-itself-research' tag as well as 'research-community' and 'research-call'**
- Can I put in multiple authors?
- Can I put in multiple categories?

**Reply by catherinet1** [reply]
Something I find helpful, which we might add to SOP, is:

- Use online tool to convert Google Doc draft to Markdown, e.g. https://gdoc2md.com/. Then paste into Github. (I cannot download Github Desktop as my laptop is too old and so am pasting directly into Github online.)

**Comment by rufuspollock** [comment]
A broader comment / thought is that we could have a workflow that took google docs directly. We worked on this (and have a diagram) in relation to converting google docs to markdown and then to brevo when building our email course here:

https://github.com/life-itself/brevo-utils

![](https://github.com/life-itself/brevo-utils/raw/main/img/workflow.svg)

**Comment by laurenwigmore** [comment]
[Walk through of how to publish a blog for LI staff here](https://drive.google.com/file/d/1iDKM95OuZTLiiqbGp1vlYiIBYj-8ZzGz/view?usp=sharing)

**Comment by rufuspollock** [comment]
## Use Google Drive Sync for collaboration rather than Github

A major clarification that came up in thread about markdown-based sites  https://github.com/orgs/life-itself/discussions/829 is that you can do collaboration without github which may be much easier https://github.com/orgs/life-itself/discussions/829#discussioncomment-7256699

> > These two points I agree with as reasons to use a MD-based approach, i.e. I think, yes we want a method where these two points are true. One obstacle for me so far has been not seeing/understanding clearly how we can easily share and easily extract the information into other workflows. I have the thought that I need to download and get familiar with multiple different new softwares and am concerned that I will run into issues using git commands to push from Obsidian into the repo etc. as I have in the past. I want to be sure that the process is efficient and that the advantages of using this method make the time investment worthwhile.
> 
> Really good point.
> 
> First, want emphasize that for collaboration / sharing there are other (perhaps easier) options than git: e.g. simply use dropbox or google drive sync and hey presto you can share amongst the team. That i think is the way better option for simple collaboration (and one of the team members can always push to github if that is wanted for other reasons e.g. for publicly publishing the site!)
> 
> I feel foolish for not having emphasized this enough before b/c i realize that github as collaboration method is the biggest blocker and actually has nothing to do with markdown approach.

**Reply by rufuspollock** [reply]
@laurenwigmore @shiweiart @nathenf @alexianetcu  i think this could make things a lot easier. I can explain more if you are interested as to how this would work.

**Comment by rufuspollock** [comment]
## Investigating tina.io

Searching for alternative more simple editing experiences i looked at tina.io

You can connect reasonably easily. However, you do have ot then go the "tina" way - see this image.

![image](https://github.com/life-itself/community/assets/180658/a5a2760b-4b92-45f3-9d4e-76aaa1f574c8)
